web/src/features/hfiCalculator/pages/HfiCalculatorPage.tsx

```diff
@@ -277,6 +277,13 @@ const HfiCalculatorPage: React.FunctionComponent = () => {
         </Container>
       ) : (
         <Container maxWidth={'xl'}>
+          {/* {hfiDailiesError ||
+            (fireCentresError && (
+              <HFIErrorAlert
+                hfiDailiesError={hfiDailiesError}
+                fireCentresError={fireCentresError}
+              />
+            ))} */}
```

Comment:
Is this commented section worth keeping in or can it be deleted?

Reply:
I'll delete it for now, it'll be going back in soon, potentially in another PR